README.md

```diff
@@ -17,6 +17,9 @@ def deps do
   ]
 end
 ```
+
+Since parts of this package are implemented in Rust, you need to have a Rust installation to compile this package. You can get one [here](https://rustup.rs/)
```

Comment:
I'm skeptical whether this would work, since every computer can have a different OpenGL implementation and we need to link against it. Doesn't seem like something that could realistically be done, since every graphics driver update may potentially require a relink.

Reply:
isn't opengl implementation provided by the drivers linked dynamically when you initialize gl?

Reply:
I honestly don't know. I've been having linking issues for a long time in the experimental stage of this project. It should be possible to link dynamically, but I think this could be a lot of work and might be out of the scope of this PR. Do you think it's OK to merge this as-is and add figuring out precompilation to the backlog?

Reply:
yes, that's definitely ok to do this in separate PR :)